app.css

```diff
@@ -1,8 +1,393 @@
 * {
   box-sizing: border-box;
   margin: 0;
+  font-family: sans-serif;
 }
 
 *::before, *::after {
   box-sizing: inherit;
 }
+
+.total {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  background-image: url(http://www.crestingwave.com/sites/default/files/splashimages/wave7.png);
+  /*filter: blur(5px);
+    -webkit-filter: blur(5px);*/
+  background-size: 100% 100%;
+  background-position: center;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  z-index: -1;
+}
+
+.header:before {
+  content: '';
+   position: absolute;
+   top: 0;
+   left: 0;
+   bottom: 0;
+   right: 0;
+   background: rgba(63, 76, 167, 0.7);
+
+}
+
+.header_navbar {
+  display:flex;
+  padding: 1rem;
+  padding-left: 15vw;
```

Comment:
Look at using a double container to center things up here instead of this padding left. This will break down and won't keep things centered up properly